ansible/roles/test/files/helpers/add_ip.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
```

Comment:
this is used by copp.yml

roles/test/tasks/copp.yml:      script: roles/test/files/helpers/add_ip.sh